docs/en_US/reference/experiment_config.rst

```diff
@@ -1,26 +1,29 @@
-===========================
-Experiment Config Reference
-===========================
+==============================
+Experiment Config V2 Reference
+==============================
 
-Notes
-=====
+A config file is needed when creating an experiment. This document describes the rules to write a config file in the V2 schema and provides some examples.
 
-1. This document list field names is ``camelCase``.
-   They need to be converted to ``snake_case`` for Python library ``nni.experiment``.
+.. Note::
 
-2. In this document type of fields are formatted as `Python type hint <https://docs.python.org/3.10/library/typing.html>`__.
-   Therefore JSON objects are called `dict` and arrays are called `list`.
+    1. This document lists field names with ``camelCase``. They need to be converted to ``snake_case`` for Python library ``nni.experiment``.
 
-.. _path:
+    2. In this document, the type of fields are formatted as `Python type hint <https://docs.python.org/3.10/library/typing.html>`__. Therefore JSON objects are called `dict` and arrays are called `list`.
 
-3. Some fields take a path to file or directory.
-   Unless otherwise noted, both absolute path and relative path are supported, and ``~`` will be expanded to home directory.
+    .. _path: 
```

Comment:
what is this .. _path: used for?

Reply:
An anchor, when quoted below, users can jump directly, like these:
```Path_ to the JSON file containing the search space```
```Specify the :ref:`directory <path>` to place log, checkpoint, metadata, and other run-time stuff.```